src/niscope/templates/session.py/ext_cal_last_temp.py.mako

```diff
@@ -0,0 +1,13 @@
+<%page args="f, config, method_template"/>\
+<%
+    '''Forwards to _cal_fetch_temp, with external calibration pre-selected.'''
+    import build.helper as helper
+%>\
+    def ${f['python_name']}(${helper.get_params_snippet(f, helper.ParameterUsageOptions.SESSION_METHOD_DECLARATION)}):
+        '''${f['python_name']}
+
+        ${helper.get_function_docstring(f, False, config, indent=8)}
+        '''
+
+        return self._cal_fetch_temperature(1)
```

Comment:
Passing 1 and 0 is not a good idea.
I believe it's possible for nimi-python to generate private enums. Right, @texasaggie97 ?

Reply:
I think you should look into how to use the same mako template for both ext and self flavors of the function. You can probably switch on something in order to send the right enum value.

Reply:
I've used an enum instead of numbers, and collapsed ext and self into single templates as suggested!